pkg/kn/commands/plugin/install.go

```diff
@@ -0,0 +1,146 @@
+// Copyright © 2024 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin
+
+import (
+	"errors"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"knative.dev/client/pkg/kn/commands"
+	"knative.dev/client/pkg/kn/config"
+)
+
+var updateFlag bool
+
+// NewPluginInstallCommand creates a new `kn plugin install` command
+func NewPluginInstallCommand(p *commands.KnParams) *cobra.Command {
+
+	var localPath string
+	cmd := &cobra.Command{
+		Use:     "install",
+		Short:   "Install a plugin from URL or local file",
+		Aliases: []string{"ls"},
```

Comment:
Why is "ls" an alias ? Doesn't sound intuitive.

Reply:
Thnaks, good catch! That's an copy paste error on my side probably. I'll remove it.